src/components/information/QuestionEditField.vue

```diff
@@ -55,5 +58,20 @@ const requiredRule = (v: unknown) => (v === undefined || v === null ? '必須項
     :items="question.options"
     :item-value="(option) => option.id"
     :item-title="(option) => option.content"
+    class="wrap-text"
   ></v-select>
 </template>
+
+<style scoped>
+.wrap-text :deep(.v-select__selection-text) {
```

Comment:
```suggestion
<style module>
.wrapText :deep(.v-select__selection-text) {
```
CSS Modules への変更をお願いしたいです！